Assets/Scripts/Game/TalkManager.cs

```diff
@@ -1056,22 +1031,21 @@ public string DirectionVector2DirectionHintString(Vector2 vecDirectionToTarget)
 
             if ((angle >= 0.0f && angle < 22.5f) || (angle >= 337.5f && angle <= 360.0f))
                 return TextManager.Instance.GetText(textDatabase, "east");
-            else if (angle >= 22.5f && angle < 67.5f)
+            if (angle >= 22.5f && angle < 67.5f)
```

Comment:
Maybe that could be simplified using some maths like in
https://github.com/Interkarma/daggerfall-unity/pull/1730/files#diff-c4ce0a36388a746d93049250a142647fR350
? Putting direction strings in an array...

Reply:
I also noticed that code block while reviewing it, that sounds like a good idea, instead of hardcoding them into if conditions.